pkg/interface/src/views/apps/links/components/comment-item.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
```

Comment:
any reason you're de-componentising this

Reply:
...an accident? I don't think this is used anywhere, looks like a merge conflict

Reply:
oki